docker/Dockerfile

```diff
@@ -33,6 +33,7 @@ ENV DOCKERDIR=./docker/
 COPY . /app/elkarbackup
 RUN set -ex; \
       cd /app/elkarbackup; \
+      rm app/config/parameters.yml; \
```

Comment:
Uxue, have you tested this without a app/config/parameters.yml file? A fresh checkout from git won't have such a file so this rm will fail? Will the docker build succeed in that situation?

Reply:
You're right, it fails if there is no parameters.yml file. I will correct that